src/screens/PinScreen.js

```diff
@@ -192,13 +192,26 @@ class PinScreen extends React.Component {
         return;
       }
 
-      const pinCorrect = cryptoUtils.checkPassword(accessData.words, pin);
+      let wordsEncryptedData = accessData.words;
+      if (!accessData.words.data) {
+        // This is from a previous version
```

Comment:
I would suggest naming the version this refers to, like: "Versions before 0.26.0 didn't have the `data` property".

I think it would make it clearer when it's OK to remove this validation down the road.

Reply:
It may never be ok to remove this validation because anyone with a version prior to v0.26.0 may upgrade to the most recent version and not having this would require the person to import the wallet from scratch